trunk/source/CDTParser/src/de/uni_freiburg/informatik/ultimate/cdt/parser/GccStaticLanguageSettingsProvider.java

```diff
@@ -262,6 +262,7 @@ public GccStaticLanguageSettingsProvider() {
 				createEntry("_stdcall", "__attribute__((__stdcall__))"),
 				createEntry("_thiscall", "__attribute__((__thiscall__))"),
 				createEntry("__builtin_va_arg(ap,type)", "*((typeof(type) *)((ap += sizeof(type)) - sizeof(type)))"),
+				createEntry("va_arg(ap,type)", "*((typeof(type) *)((ap += sizeof(type)) - sizeof(type)))"),
```

Comment:
Could you please add comments here to distinguish which entries (such as this one) have been added by us manually (and for what reason), as opposed to the remaining ones which, according to the class documentation, have at some point been generated automatically?